packages/components/src/context-actions/ContextActionUtils.ts

```diff
@@ -116,6 +116,11 @@ class ContextActionUtils {
    * @returns Promise Resolved on success, rejected on failure
    */
   static copyToClipboard(text: string): Promise<void> {
+    const { clipboard } = navigator;
```

Comment:
This works - I think you could also write this function as an async function and it would be a little cleaner:

```typescript
  static async copyToClipboard(text: string): Promise<void> {
    try {
      await navigator.clipboard.writeText(text);
    } catch (e) {
      ContextActionUtils.copyToClipboardExecCommand(text)
    }
  }
```